dm/syncer/binlog_locations.go

```diff
@@ -33,7 +34,8 @@ type locationRecorder struct {
 	//       |                       |
 	// curStartLocation        curEndLocation
 	// there may be more events between curStartLocation and curEndLocation due to the limitation of binlog or
-	// implementation of DM, but those events should always belong to one transaction.
+	// implementation of DM, but in such scenario, those events should always belong to one transaction.
+	// When curStartLocation is equal to curEndLocation, it means current event is not a data change.
```

Comment:
just a question: why `When` is capitalize

Reply:
English grammar: capitalize the first letter of a sentence